pkg/flow/database.go

```diff
@@ -786,6 +866,13 @@ type instvarData struct {
 	vdata *ent.VarData
 }
 
+func (ivd *instvarData) tags() map[string]string {
+	tags := ivd.instData.tags()
+	tags["id-vref"] = ivd.vref.Name
```

Comment:
I think we need to write a document outlining every tag we intend to use. And as part of writing that document, we need to come up with friendlier keys.